src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -451,12 +458,14 @@ public function timeout(int $seconds)
      * @param  int  $times
      * @param  int  $sleep
      * @param  callable|null  $when
+     * @param  bool  $throw
      * @return $this
      */
-    public function retry(int $times, int $sleep = 0, ?callable $when = null)
+    public function retry(int $times, int $sleep = 0, ?callable $when = null, bool $throw = true)
```

Comment:
We can't change the method signature on a stable release unfortunately.

Reply:
Should I submit this to master then?

Reply:
Yeah, if we can't solve it differently then that would be best.

Reply:
Okay. I’ll submit it for master.